### PR TITLE
Add Use Goals, Questions, and Metrics section

### PR DESCRIPTION
**Pull request by jeffabailey**

### CODE REVIEW

File: SUMMARY.md [hunk 1]
```diff
@@ -14,6 +14,13 @@
   * [Authors and Reviewers](infrastructure/authors.md)
 * [What, When and How to Measure](measuring/introduction.md)
   * [Goals using Metrics](measuring/goals.md)
+    * [Use Goals, Questions, and Metrics](measuring/use\_gqm.md)
+      * [Goals](use\_gqm/goals/README.md)
```
Comment (rrrutledge) [inline SUMMARY.md:18]: I wonder if there's a way to auto-generate this part

File: measuring/use_gqm.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Use Goals, Questions, and Metrics
+
+*An overview of the GQM catalog.*
+
+```mermaid
+graph LR;
+reduce-duplication[Reduce Duplication] --> who-uses[Who Uses];
+who-uses --> usage-count[Usage Count];
+usage-count --> reduce-duplication[Reduce Duplication];
```
Comment (rrrutledge) [inline measuring/use_gqm.md:9]:
```suggestion
```
Take this out - direction is one-way only - G->Q->M

**Review by rrrutledge — COMMENTED** [review]
Thank you!

File: SUMMARY.md (hunk 1 above)
Comment (jeffabailey) [reply SUMMARY.md:18]: Yes, and we will!

**Review by rrrutledge — APPROVED** [review]
Yes ‼️

**Pull request merged**